src/sortedvector.jl

```diff
@@ -22,8 +22,9 @@ SortedVector([1, 2, 3, 4, 5])
 struct SortedVector{T<:Number} <: AbstractVector{T}
     data::Vector{T}
 
-    function SortedVector{T}(data::AbstractVector{T}) where {T}
-        return new{T}(convert(Vector{T}, data))
+    function SortedVector{T}(data::AbstractVector{T}; sorted=false) where {T}
+        sorted_data = ifelse(sorted, data, sort(data))
+        return new{T}(convert(Vector{T}, sorted_data))
     end
```

Comment:
we could also keep the non-parametrized version of this constructor

Reply:
I removed it because it never got called. Where would you need it?

Reply:
Not sure